src/FubuTransportation/Subscriptions/SubscriptionActivator.cs

```diff
@@ -17,18 +16,23 @@ public class SubscriptionActivator : IActivator
         private readonly ISubscriptionCache _cache;
         private readonly IEnumerable<ISubscriptionRequirement> _requirements;
         private readonly ChannelGraph _graph;
+        private readonly TransportSettings _settings;
 
-        public SubscriptionActivator(ISubscriptionRepository repository, IEnvelopeSender sender, ISubscriptionCache cache, IEnumerable<ISubscriptionRequirement> requirements, ChannelGraph graph)
+        public SubscriptionActivator(ISubscriptionRepository repository, IEnvelopeSender sender, ISubscriptionCache cache, IEnumerable<ISubscriptionRequirement> requirements, ChannelGraph graph,
+            TransportSettings settings)
         {
             _repository = repository;
             _sender = sender;
             _cache = cache;
             _requirements = requirements;
             _graph = graph;
+            _settings = settings;
         }
 
         public void Activate(IEnumerable<IPackageInfo> packages, IPackageLog log)
         {
+            if(_settings.Disabled) return;

```

Comment:
I don't think you need TransportSettings in this activator if you're handling it in FubuTransportationActivator.